### PR TITLE
fixed prettier formatting

### PR DESCRIPTION
**Pull request by tumms2021389**
- Fixed prettier configurations file
- Added prettier command in package.json
- Ran the prettier through all files

### CODE REVIEW

**Comment by ghoshArnab** [comment]
Nice 👍 
Please add a prettier check script - "prettier-check": "prettier --color -c ." and keep as part of merge check also alongside lint.

**Review by ghoshArnab — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -23,8 +23,12 @@
     "copy-local-component-map-to-lib": "shx cp ./packages/react-sdk-components/sdk-local-component-map.js ./packages/react-sdk-components/lib",
     "copy-types-to-lib": "shx mkdir -p ./packages/react-sdk-components/lib/types && shx cp -r ./packages/react-sdk-components/src/types ./packages/react-sdk-components/lib",
     "edit-pega-components-map": "node scripts/edit-pega-components-map-in-lib.js",
-    "lint": "npx eslint \"packages/react-sdk-components/src/**\"",
-    "lint-errors-only": "npx eslint \"packages/react-sdk-components/src/**\" --quiet",
+    "lint": "run-p -cl lint:*",
+    "lint:es": "eslint --color --cache --cache-location node_modules/.cache/eslint/ \"packages/react-sdk-components/src/**\"",
+    "lint:format": "prettier --loglevel warn -c .",
```
Comment (vishalshrm539) [inline package.json:28]: `--loglevel` => `--log-level`

File: package.json [hunk 2]
```diff
@@ -23,8 +23,12 @@
...
+    "fix": "run-s -cl fix:*",
+    "fix:es": "eslint --color --fix --cache --cache-location node_modules/.cache/eslint/ \"packages/react-sdk-components/src/**\"",
+    "fix:format": "prettier --loglevel warn -w .",
```
Comment (vishalshrm539) [inline package.json:31]: `--loglevel` => `--log-level`

**Review by vishalshrm539 — CHANGES_REQUESTED** [review]
Added a couple of comments in the package.json

File: .prettierignore [hunk 3]
```diff
@@ -0,0 +1,7 @@
+**/.git
+**/.github
+**/.vscode
+**/.svn
+**/.hg
+**/*.md
+**/assets
```
Comment (vishalshrm539) [inline .prettierignore:7]: Shoudn't node_modules be added here?

Comment (tumms2021389) [reply .prettierignore:7]: Not needed, prettier will ignore node_module by default

**Pull request merged**